### PR TITLE
create biosketch for nick (#1)

### PR DESCRIPTION
**Pull request by nrbgt**
Very simple at first!

### CODE REVIEW

**Review by zchen458 — APPROVED** [review]
Nice Work!

File: _posts/2020-05-13-nick-bollweg.markdown [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
```
Comment (sajadkhodadadian) [inline _posts/2020-05-13-nick-bollweg.markdown:2]: good

**Review by nivaranavat — COMMENTED** [review]
nothing wrong!

**Comment by David-davidlxl** [comment]
Great job!

**Comment by smv30** [comment]
that extra comma was critical! Everything else looks good

**Review by smileey — APPROVED** [review]

**Pull request merged**